### PR TITLE
Bigcoinstorm

### PR DESCRIPTION
**Pull request by waprin**
This adds an example of using Storm to take data from the Coinbase websocket API and insert it into Cloud Bigtable.

It's still a bit rough around the edges, I probably need to cleanup some more before I merge into master.

### CODE REVIEW

File: java/storm/src/main/java/com/example/bigtable/storm/bolts/BigtableBolt.java [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+
+            String rowKey = data.getType() + "_" + ts;
+            String columnFamily = "bc";
+
+            String column = "data";
+
+            Table table;
+            synchronized (connection) {
+                try {
+                    table = connection.getTable(TableName.valueOf
```
Comment (sduskis) [inline java/storm/src/main/java/com/example/bigtable/storm/bolts/BigtableBolt.java]: Can you do this in prepare?

Does BufferedMutator make sense here?

File: java/storm/src/main/java/com/example/bigtable/storm/bolts/BigtableBolt.java [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+    }
+
+    /**
+     * Sets up our connection to Google Cloud Bigtable.
+     */
+    @Override
+    public void prepare(Map conf, TopologyContext context, OutputCollector collector) {
+        _collector = collector;
+        try {
+            this.connection = ConnectionFactory.createConnection();
```
Comment (sduskis) [inline java/storm/src/main/java/com/example/bigtable/storm/bolts/BigtableBolt.java]: This connection needs to be closed somewhere.

**Comment by lesv** [comment]
Have you done Solomon's changes yet?

File: java/storm/pom.xml [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+      <version>0.9.4</version>
+      <!-- keep storm out of the jar-with-dependencies -->
+      <scope>provided</scope>
+    </dependency>
+    <!-- for now -->
+    <!-- mvn install:install-file -Dfile=bigtable-hbase-0.1.4.jar -DgroupId=bigtable-client -DartifactId=bigtable-hbase -Dversion=0.1.4 -Dpackaging=jar -DgeneratePom=true -->
+    <dependency>
+      <groupId>bigtable-client</groupId>
+      <artifactId>bigtable-hbase</artifactId>
+      <version>0.1.4</version>
```
Comment (lesv) [inline java/storm/pom.xml]: 0.1.5 is what we've released.

**Comment by lesv** [comment]
Please do Solomon's changes, and my one liner, then lgtm

**Comment by lesv** [comment]
Also, it would be good to add in CheckStyle in next iteration (if any).

**Comment by lesv** [comment]
LGTM

**Pull request merged**